app/models/labware_creators/plate_split_to_tube_racks.rb

```diff
@@ -240,17 +240,17 @@ def must_have_sufficient_tubes_in_rack_files
 
     # Checks the files parsed correctly
     def files_correctly_parsed?
-      return contingency_file_correctly_parsed? if require_contingency_tubes_only?
+      return contingency_file_valid? if require_contingency_tubes_only?
 
-      contingency_file_correctly_parsed? && sequencing_file_correctly_parsed?
+      contingency_file_valid? && sequencing_file_valid?
     end
 
-    def sequencing_file_correctly_parsed?
-      sequencing_file.present? && sequencing_csv_file.correctly_parsed?
+    def sequencing_file_valid?
+      sequencing_file.present? && sequencing_csv_file&.correctly_parsed? && sequencing_csv_file&.valid?
```

Comment:
I think you can remove the check for `correctly_parsed?` here as it is part of the file validations so should be checked when `valid?` is called.

Reply:
Ok will do thx